packages/api/tests/lambdas/test-bulk-operation.js

```diff
@@ -374,38 +455,70 @@ test.serial('bulk operation BULK_GRANULE_DELETE deletes granule IDs returned by
   t.deepEqual(
     deletedGranules.sort(),
     [
-      granules[0].granuleId,
-      granules[1].granuleId,
+      granules[0].newDynamoGranule.granuleId,
+      granules[1].newDynamoGranule.granuleId,
     ].sort()
   );
+
+  const s3Buckets = granules[0].s3Buckets;
+  t.teardown(() => deleteS3Buckets([
+    s3Buckets.protected.name,
+    s3Buckets.public.name,
+  ]));
 });
 
 test.serial('bulk operation BULK_GRANULE_DELETE does not fail on published granules if payload.forceRemoveFromCmr is true', async (t) => {
-  const granuleModel = new Granule();
   const granules = await Promise.all([
-    granuleModel.create(fakeGranuleFactoryV2({ published: true })),
-    granuleModel.create(fakeGranuleFactoryV2({ published: true })),
+    createGranuleAndFiles({ dbClient: t.context.knex, published: true }),
+    createGranuleAndFiles({ dbClient: t.context.knex, published: true }),
   ]);
 
-  const { deletedGranules } = await bulkOperation.handler({
-    type: 'BULK_GRANULE_DELETE',
-    envVars,
-    payload: {
-      ids: [
-        granules[0].granuleId,
-        granules[1].granuleId,
-      ],
-      forceRemoveFromCmr: true,
-    },
-  });
+  sinon.stub(
```

Comment:
Yup, totally agree. In hindsight this is a bit messy. I tossed a question in IX and Mark had a good suggestion. Rather than mocking `unpublishGranule` here, it might be better to test the `bulkGranuleDelete` logic separate from the Lambda handler. Then, if we do that, we can pass a mock of `unpublishGranule` directly to `bulkGranuleDelete`.

That conversation is here:

https://eosdis.slack.com/archives/CLBC362EQ/p1614625915373300

And the change is here:

https://github.com/nasa/cumulus/pull/2108/commits/4ad56143a43c5f26f78f5bd873e547f8bd00ca43

See what ya think!